chain-impl-mockchain/src/multiverse.rs

```diff
@@ -208,8 +208,8 @@ mod test {
     };
 
     use chain_addr::Discrimination;
-    use chain_core::property::Block as _;
-    use chain_storage::BlockStoreBuilder;
+    use chain_core::property::{self, Block as _, BlockId as _, Deserialize, Serialize};
```

Comment:
can we remove this deps?

Reply:
Yes, looks like it is only used in tests. Will do.